TL/24_simple_design_patterns_singleton_page_object/Design_Patterns_README.md

```diff
@@ -0,0 +1,137 @@
+# Design Patterns
+***
+
+## Contents
+1. [Simple Design Patterns.](#simple-design-patterns)
```

Comment:
remove ~Simple~ or change to **About Design Patterns**

Reply:
done